x/protorev/keeper/msg_server.go

```diff
@@ -71,6 +71,15 @@ func (m MsgServer) SetMaxPoolPointsPerTx(c context.Context, msg *types.MsgSetMax
 		return nil, err
 	}
 
+	maxPointsPerBlock, err := m.k.GetMaxPointsPerBlock(ctx)
```

Comment:
Non-blocking but just FYI for future contributions that it is beneficial to separate state-breaking changes from non-state-breaking so that we can release non-state-breaks sooner.

We have docs for this here:
https://github.com/osmosis-labs/osmosis/blob/d945e6230d042d26c700717b2373cd6d5976e48b/CONTRIBUTING.md?plain=1#L365

If we only had CLI changes in this PR, we could release them in the v15 line. Since there are message server changes, this will have to wait till v16.